tensorboard/encode_png_benchmark.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Benchmarks for the `tensorboard.util.encode_png` function.
+
+Here are the results of running this benchmark on a workstation running
+Ubuntu 14.04 with an Intel(R) Xeon(R) CPU E5-1650 v4 @ 3.60GHz:
+
+    THREADS  TOTAL_TIME  UNIT_TIME  SPEEDUP  PARALLELISM
```

Comment:
Okay. I just re-ran with your build, and the results are essentially the same. (The average time increased by 0.005s, or a factor of 1.0023.) Updated to use those times in any case.